src/psyclone/tests/domain/lfric/lfric_config_test.py

```diff
@@ -86,86 +90,158 @@ def clear_config_instance():
     Config._instance = None
 
 
-@pytest.mark.parametrize("option", ["access_mapping", "COMPUTE_ANNEXED_DOFS",
-                                    "default_kind", "RUN_TIME_CHECKS"])
+def config(config_file, content):
+    ''' A utility function that creates and populates a temporary
+    PSyclone configuration file for testing purposes.
+
+    :param config_file: local path to the temporary configuration file.
+    :type config: :py:class:`py._path.local.LocalPath`
+    :param content: the entry for the temporary configuration file.
+    :type content: str
+
+    :returns: a test Config instance.
+    :rtype: :py:class:`psyclone.configuration.Config`
+
+    '''
+    with config_file.open(mode="w") as new_cfg:
+        new_cfg.write(content)
+        new_cfg.close()
+        return Config()
```

Comment:
Please change this function in the same way as suggested for `config_test.py`

Reply:
Done as above.